### PR TITLE
[14.0][MIG] sale_stock_picking_note from 13.0

### PR DESCRIPTION
**Pull request by acsonefho**

### CODE REVIEW

File: sale_stock_picking_note/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _get_new_picking_values(self):
+        """
+        Inherit to fill the note (on the picking) with value of picking_note on the SO
+        """
+        values = super()._get_new_picking_values()
+        sale_note = self.sale_line_id.order_id.picking_note
```
Comment (rousseldenis) [inline sale_stock_picking_note/models/stock_picking.py:15]: @acsonefho I think this method is multi. Can you check ?

Comment (rousseldenis) [reply sale_stock_picking_note/models/stock_picking.py:15]: @acsonefho

**Comment by pedrobaeza** [comment]
Can you please include #1698 ?

**Comment by lmignon** [comment]
ping @acsonefho can you take into account the comments to prevent this RP from going stale. Thank you for your work.

**Comment by victoralmau** [comment]
Superseed by: https://github.com/OCA/sale-workflow/pull/1769

**Comment by rousseldenis** [comment]
@acsonefho Could you attend @pedrobaeza comment in order to merge this ?

**Comment by pedrobaeza** [comment]
As no further notice, closing in favor of #1769

**Pull request closed without merging**